### DISCUSSION TITLE
Add folders to the node tree

### DISCUSSION THREAD

**Discussion by mayrinck** [tsujan/FeatherNotes#362] [Ideas]
Hey there guys, I came across this project a few days ago and I've been testing it out. So far, I'm liking it. Currently, this project is one of the best alternatives to Onenote. However, a few enhancements could be made to help organize the node tree and improve the overall experience.

The idea is to introduce **folders**, much like sections in Onenote. Sections help users organize a specific set of pages under a particular theme or label. This mirrors how Onenote manages sections and section groups with a simple yet familiar concept.

With folders, users can create them to separate and organize nodes within the node tree. Currently, something similar can be achieved by making nodes children of another node. However, this method always leaves a parent node empty or without content if the user just desires that to be an organizer node and not a functional node (with content inside).

Implementing this idea would significantly enhance the user experience, and I believe it should be considered for future updates.

Thanks in advance.

**Comment by tsujan** [comment]
> Currently, something similar can be achieved by making nodes children of another node

Yes. There's no reason to do more, especially because the user can add a folder icon (or any other icon of his/her choice) to nodes which he considers to be "organizers". Adding codes to remove features (in this case, the ability to add notes to  "organizers") doesn't seem good to me.

**Reply by mayrinck** [reply]
I understand your position, but this is not a way to "remove features", this is a way to make things more logical from the user's perspective IMO. A Lock feature would work, as the user will be able to have a note where no changes can be made (and revert it by unlocking if desired)
